lib/libc/picolibc/CMakeLists.txt

```diff
@@ -22,22 +21,6 @@ zephyr_compile_definitions(__LINUX_ERRNO_EXTENSIONS__)
 if(NOT CONFIG_PICOLIBC_USE_MODULE)
 
   # Use picolibc provided with the toolchain
-
-  # Create a C import library to apply dependencies
-
-  add_library(c STATIC IMPORTED GLOBAL)
```

Comment:
It sounds like I can use `IMPORTED_LIBNAME` here and get rid of the file hack?

Reply:
I still think it's correct to have these libraries created by the libc CMakeLists.txt and then pass the list to the kernel CMakeLists.txt for later dependency adding.